pjmedia/include/pjmedia/conference.h

```diff
@@ -104,6 +118,87 @@ enum pjmedia_conf_option
                                      based.                                 */
 };
 
+/**
+ * This structure specifies the conference bridge creation parameters.
+ */
+typedef struct pjmedia_conf_param
+{
+    /**
+     * Maximum number of slots/ports to be created in
+     * the bridge. Note that the bridge internally uses
+     * one port for the sound device, so the actual 
+     * maximum number of ports will be less one than
+     * this value.
+     */
+    unsigned max_slots;
+
+    /**
+     * Set the sampling rate of the bridge. This value
+     * is also used to set the sampling rate of the
+     * sound device.
+     */
+    unsigned sampling_rate;
+
+    /**
+     * Number of channels in the PCM stream. Normally
+     * the value will be 1 for mono, but application may
+     * specify a value of 2 for stereo. Note that all
+     * ports that will be connected to the bridge MUST 
+     * have the same number of channels as the bridge.
+     */
+    unsigned channel_count;
+
+    /**
+     * Set the number of samples per frame. This value
+     * is also used to set the sound device.
+     */
+    unsigned samples_per_frame;
+
+    /**
+     * Set the number of bits per sample. This value
+     * is also used to set the sound device. Currently
+     * only 16bit per sample is supported.
+     */
+    unsigned bits_per_sample;
+
+    /**
+     * Bitmask options to be set for the bridge. The
+     * options are constructed from #pjmedia_conf_option
+     * enumeration.
+     * The default value is zero.
+     */
+    unsigned options;
+
+    /** 
+     * The number of worker threads to use.
```

Comment:
Add doc that the value will not be used for switchboard.

Also in pjsua and pjsua2.